app/assets/stylesheets/media-box.scss

```diff
@@ -1,4 +1,20 @@
-.media { margin: 10px; }
+.media {
+  margin: 10px;
+  position: relative;
+
+  .gallery-picture img { margin-bottom: 20px; }
+
+  .card-footer .footer-container {  font-size: 13px; }
+
+  .card-footer span { float: right; }
```

Comment:
Again, you could move these card-footer / gallery-picture specific lines to a dedicated card-footer file.